### ISSUE TITLE
Blob token keyboard accessibility has no keyboard control to open tooltip (and therefor find references too)

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#44464]
You can trigger Go to definition by activating the link, but there's no way to open the tooltip with the keyboard.

There should be some keyboard shortcut that opens the tooltip and focuses it.

Alternatively, the tooltip could be opened automatically on focus and associated with `aria-describedby`, considering that it is also opened on hover. This would be consistent with how other tooltips work and would be better for discoverability.

**Comment by felixfbecker** [comment]
@quinnkeast thoughts on this?

**Comment by lrhacker** [comment]
This requirement is captured in #42421 and is also already included in #44698 (and being discussed there). Closing as a duplicate.
